src/conf_macros.m4

```diff
@@ -661,6 +661,23 @@ AC_DEFUN([WITH_EXTENDED_ENUMERATION_SUPPORT],
     AM_CONDITIONAL([BUILD_EXTENDED_ENUMERATION_SUPPORT], [test x"$with_extended_enumeration_support" = xyes])
   ])
 
+AC_DEFUN([WITH_ALLOW_REMOTE_DOMAIN_LOCAL_GROUPS],
+  [ AC_ARG_WITH([allow-remote-domain-local-groups],
+                [AC_HELP_STRING([--with-allow-remote-domain-local-groups],
+                                [Whether to build support of ad_allow_remote_domain_local_groups
+                                 sssd.conf option [no].]
```

Comment:
Hi,

I would like to suggest to mention here as well that the option is deprecated.

bye,
Sumit

Reply:
Fixed (and rebased).